src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformerTest.php

```diff
@@ -17,10 +17,8 @@
 
 class IntegerToLocalizedStringTransformerTest extends TestCase
 {
-    protected function setUp()
+    protected function tearDown()
```

Comment:
not doing it on `setUp` means that the first test of this TestCase will run with whatever previous default is set. This might cause other issues.

the proper fix should rather be to save the existing default in `setUp` and then change it to whatever we need for this test, and then restoring it in `tearDown`

Reply:
see #31448

Reply:
#31448 merged up to master. @ro0NL Can you rebase this one?